### PR TITLE
address #315 and drive by global bindings

### PR DESCRIPTION
**Pull request by JaseZiv**

### CODE REVIEW

File: R/get_advanced_match_stats.R [hunk 1]
```diff
@@ -147,7 +147,12 @@ fb_advanced_match_stats <- function(match_url, stat_type, team_or_player, time_p
           }
 
 
-          stat_df_output <- cbind(match_report, stat_df_output)
+          stat_df_output <- dplyr::bind_cols(match_report, stat_df_output)
```
Comment (JaseZiv) [inline R/get_advanced_match_stats.R:150]: `dplyr::bind_cols()` allows us to not fail at this point if there's missing data for matches

File: R/internals.R [hunk 2]
```diff
@@ -96,20 +96,20 @@
   names(stat_df) <- new_names
   stat_df <- stat_df[-1, ]
 
-  stat_df <- stat_df %>% dplyr::select(-Matches)
+  stat_df <- stat_df %>% dplyr::select(-.data[["Matches"]])
```
Comment (JaseZiv) [inline R/internals.R:99]: Drive by fixes of previous PR with global bindings

**Review by tonyelhabr — APPROVED** [review]
thanks for this cleanup!

at some point we'll (unfortunately) need to replace all of the `.data` stuff since i think that syntax may be deprecated at some point in the future.

**Comment by JaseZiv** [comment]
> thanks for this cleanup!
> 
> 
> 
> at some point we'll (unfortunately) need to replace all of the `.data` stuff since i think that syntax may be deprecated at some point in the future.

Yeah I was thinking that as I was doing it, then thought I'd keep this as uniform as possible so when we do address it, might make our lives a little easier looking for consistent patterns

**Pull request merged**